cmd/gpud/command/list_plugins.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli"
+
+	v1 "github.com/leptonai/gpud/client/v1"
+	"github.com/leptonai/gpud/pkg/log"
+)
+
+// cmdListPlugins implements the list-plugins command
+func cmdListPlugins(c *cli.Context) error {
+	// Set up logging
+	zapLvl, err := log.ParseLogLevel(logLevel)
+	if err != nil {
+		return err
+	}
+	log.Logger = log.CreateLogger(zapLvl, logFile)
+
+	// Create a context with timeout
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	defer cancel()
+
+	// Get custom plugins
+	plugins, err := v1.GetCustomPlugins(ctx, "http://localhost:8080")
```

Comment:
We can use the similar pattern as https://github.com/leptonai/gpud/blob/83b26323bf8b91aae7ceac41562600527ee5b6b3/cmd/gpud/command/status.go#L75-L83 to use the default endpoint?